app/controllers/outings_controller.rb

```diff
@@ -0,0 +1,16 @@
+class OutingsController < ApplicationController
+
+  def index
+    @search = Outing.search(params[:q])
+    @outings = @search.result
+    @q = Outing.ransack(params[:q])
+    @outings = @q.result.includes(:politician)
```

Comment:
@sallymacnicholas It has to do with the weirdness involved with the Ransack Gem.